docs/modules/operation/pages/service-assurance/monitors/NrpeMonitor.adoc

```diff
@@ -1,8 +1,8 @@
 
 = NrpeMonitor
 
-This monitor allows to test plugins and checks running on the #link:http://exchange.nagios.org/directory/Addons/Monitoring-Agents/NRPE--2D-Nagios-Remote-Plugin-Executor/details[Nagios Remote Plugin Executor]# (NRPE) framework.
-The monitor allows to test the status output of any available check command executed by NRPE.
+Lets you test plugins and checks running on the #link:http://exchange.nagios.org/directory/Addons/Monitoring-Agents/NRPE--2D-Nagios-Remote-Plugin-Executor/details[Nagios Remote Plugin Executor]# (NRPE) framework.
+The monitor tests the status output of any available check command executed by NRPE.
 Between {page-component-title} and Nagios are some conceptional differences.
```

Comment:
Move the comparison to a sperate paragraph.  The sentences following are missing some commas.
```suggestion

{page-component-title} and Nagios have several conceptional differences.
```